### PR TITLE
Module 22

### PR DESCRIPTION
**Pull request by angieVa**

### CODE REVIEW

File: test/locators.spec.ts [hunk 1]
```diff
@@ -27,11 +27,13 @@ describe('Given a practice form page', () => {
...
           'Wait Commands',
           'WebElement Commands'],
+        picture: './resources/calcifer.jpg',
       });
-      await personalInformationPage.submitForm();
     });
 
-    it('Then should have a title', async () => {
+    it('Then the image should have been uploaded and should have a title', async () => {
+      expect(await personalInformationPage.getProfilePictureValue()).toEqual('calcifer.jpg');
```
Comment (germandavid85) [inline test/locators.spec.ts:35]: I don't see the calcifieer.jpg file as part of this PR
image

File: test/locators.spec.ts [hunk 2]
```diff
@@ -27,11 +27,13 @@ describe('Given a practice form page', () => {
...
+      await personalInformationPage.submitForm();
```
Comment (germandavid85) [inline test/locators.spec.ts]: `await personalInformationPage.submitForm();` should not be within the `it`

File: src/page/personal-information.page.ts [hunk 3]
```diff
@@ -22,8 +25,9 @@ export class PersonalInformationPage {
   private toolsInput: ElementArrayFinder; */
   private continentSelector: ElementFinder;
   private commandsSelector: ElementFinder;
+  private pictureInput: ElementFinder;
 
-  //button y confirmation
+  //button y confirmation BORRAR COMMENTS Y CAMBIAR LOCAL CONFIG
```
Comment (germandavid85) [inline src/page/personal-information.page.ts]: is this comment needed?

**Review by germandavid85 — CHANGES_REQUESTED** [review]

File: test/locators.spec.ts (hunk 1 above)
Comment (angieVa) [reply test/locators.spec.ts:35]: image

**Review by germandavid85 — APPROVED** [review]

**Pull request merged**